### DISCUSSION TITLE
Hypseus-Singe?

### DISCUSSION THREAD

**Discussion by Gardhul** [forkymcforkface/RGBPi-Extra#2] [Q&A]
Hi, in the list of added Emulators appears "Hypseus-Singe" with the note: "Included in ports from RGBPi OS official".
What does it mean? After applied the patch I cannot see any particular folder inside /roms or /ports related to Hypseus Singe. Is it someway hidden but actually working? Is it someway supported natively by RGBPi OS? I don't have any clue to make it works...

**Comment by forkymcforkface** [comment]
An official port will be posted to official Telegram ports soon. I just submitted to the rgbpi peeps Vs adding it on rgbpi-extra, .

**Comment by forkymcforkface** [comment]
It's in the official RGBpi ports

**Comment by Gardhul** [comment]
Fantastic, thank you!

Paolo De Angelis

Il ven 31 mag 2024, 07:02 Kev ***@***.***> ha scritto:
